capabilityquerier/cmd/main.go

```diff
@@ -86,14 +86,48 @@ func queryEndpoints(message []byte, args *map[string]interface{}) error {
 	return nil
 }
 
-func main() {
-	err := config.SetupConfig()
-	helpers.FailOnError("", err)
+// queryEndpointsCapabilityStatement gets an endpoint from the queue message and queries it to get the Capability Statement.
+// This function is expected to be called by the lanternmq ProcessMessages function.
+// parameter message:  the queue message that is being processed by this function, which is just an endpoint.
+// parameter args:     expected to be a map of the string "queryArgs" to the above queryArgs struct. It is formatted
+// 					   this way because queue processing is generalized.
+func queryEndpointsVersionsOperation(message []byte, args *map[string]interface{}) error {
```

Comment:
You could combine this with queryEndpointsCapabilityStatement, but queryEndpointsCapabilityStatement does have a few extra lines so I'm okay with them being separate if you want

Reply:
fair enough!